### ISSUE TITLE
kopf fails to run on Windows

### ISSUE THREAD

**Issue by damc-dev** [zalando-incubator/kopf#219]
## Expected Behavior
kopf should run example

## Actual Behavior
Fails with NotImplementedError from asyncio
```
[2019-11-05 16:05:13,664] kopf.clients.auth    [DEBUG   ] Pykube is configured via kubeconfig file.
[2019-11-05 16:05:13,737] kopf.clients.auth    [DEBUG   ] Client is configured via kubeconfig file.
[2019-11-05 16:05:13,808] kopf.engines.peering [WARNING ] Default peering object not found, falling back to the standalone mode.
Traceback (most recent call last):
  File "c:\python37\lib\runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "c:\python37\lib\runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "C:\Python37\Scripts\kopf.exe\__main__.py", line 7, in <module>
  File "c:\python37\lib\site-packages\click\core.py", line 764, in __call__
    return self.main(*args, **kwargs)
  File "c:\python37\lib\site-packages\click\core.py", line 717, in main
    rv = self.invoke(ctx)
  File "c:\python37\lib\site-packages\click\core.py", line 1137, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "c:\python37\lib\site-packages\click\core.py", line 956, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "c:\python37\lib\site-packages\click\core.py", line 555, in invoke
    return callback(*args, **kwargs)
  File "c:\python37\lib\site-packages\kopf\cli.py", line 30, in wrapper
    return fn(*args, **kwargs)
  File "c:\python37\lib\site-packages\kopf\cli.py", line 63, in run
    peering_name=peering_name,
  File "c:\python37\lib\site-packages\kopf\reactor\running.py", line 44, in run
    peering_name=peering_name,
  File "c:\python37\lib\asyncio\base_events.py", line 579, in run_until_complete
    return future.result()
  File "c:\python37\lib\site-packages\kopf\reactor\running.py", line 77, in operator
    ready_flag=ready_flag,
  File "c:\python37\lib\site-packages\kopf\reactor\running.py", line 157, in spawn_tasks
    loop.add_signal_handler(signal.SIGINT, signal_flag.set_result, signal.SIGINT)
  File "c:\python37\lib\asyncio\events.py", line 540, in add_signal_handler
    raise NotImplementedError
NotImplementedError
```

## Steps to Reproduce the Problem

  1. Follow sample problem steps https://kopf.readthedocs.io/en/latest/walkthrough/problem/
  2. Create operator stub out create function in ephermal.py
```
 import kopf

 @kopf.on.create('zalando.org', 'v1', 'ephemeralvolumeclaims')
 def create_fn(body, **kwargs):
     print(f"A handler is called with body: {body}")
```
  3. execute command `kopf run ephemeral.py --verbose`

## Specifications

  - Platform:  Microsoft Windows 10 Enterprise  10.0.17134 N/A Build 17134
  - Kubernetes version: "v1.14.7
  - Python version:  3.7.4
  - Python packages installed: 
```
aiohttp==3.6.2
aiojobs==0.2.2
async-timeout==3.0.1
attrs==19.3.0
cachetools==3.1.1
certifi==2019.6.16
chardet==3.0.4
Click==7.0
colorama==0.4.1
cycler==0.10.0
docker==4.1.0
Flask==1.1.1
gitdb2==2.0.6
GitPython==3.0.3
google-auth==1.7.0
idna==2.8
iso8601==0.1.12
itsdangerous==1.1.0
Jinja2==2.10.3
kiwisolver==1.1.0
kopf==0.22
kubernetes==10.0.1
MarkupSafe==1.1.1
matplotlib==3.1.1
multidict==4.5.2
numpy==1.17.2
oauthlib==3.1.0
pip==19.3.1
pyasn1==0.4.7
pyasn1-modules==0.2.7
pykube-ng==19.10.0
pyparsing==2.4.2
pypiwin32==223
python-dateutil==2.8.0
pywin32==225
PyYAML==5.1.2
redis==3.3.11
requests==2.22.0
requests-oauthlib==1.2.0
rfc3339==6.1
rsa==4.0
setuptools==40.8.0
six==1.12.0
smmap2==2.0.5
urllib3==1.25.3
websocket-client==0.56.0
Werkzeug==0.16.0
yarl==1.3.0
```

**Comment by damc-dev** [comment]
It appears that the add_signal_handler method is not implemented for Windows  https://docs.python.org/3/library/asyncio-eventloop.html#asyncio.loop.add_signal_handler

A potential workaround is to just catch the NotImplementedError to handle the case of Windows.  

I will work on putting together a PR.

**Comment by nolar** [comment]
Merged and pre-released as `kopf==0.23rc1`. Will be normally released as 0.23 in few days since now. See the [Kopf 0.23rc1 Release Notes](https://github.com/zalando-incubator/kopf/releases/tag/0.23rc1).
